src/test/java/org/gridsuite/study/server/service/client/dynamicsimulation/DynamicSimulationClientTest.java

```diff
@@ -279,4 +279,34 @@ public void testDeleteResult() {
         // check result
         assertTrue(true);
     }
+
+    @Test
+    public void testResultCount() throws JsonProcessingException {
+
+        // configure mock server response for test result count - supervision/results-count
+        String resultCountEndPointUrl = UrlUtil.buildEndPointUrl("", API_VERSION, DYNAMIC_SIMULATION_END_POINT_RESULT_COUNT);
+        wireMockServer.stubFor(WireMock.get(WireMock.urlMatching(resultCountEndPointUrl))
+            .willReturn(WireMock.ok()
+                    .withBody(objectMapper.writeValueAsString(1))
+                    .withHeader("Content-Type", "application/json; charset=utf-8")
+            ));
+        Integer resultCount = dynamicSimulationClient.getResultsCount();
+
+        // check result
+        assertEquals(1, resultCount.intValue());
+    }
+
+    @Test
+    public void testDeleteResults() {
+
+        // configure mock server response for test delete all results - results/
+        String resultEndPointUrl = UrlUtil.buildEndPointUrl("", API_VERSION, DYNAMIC_SIMULATION_END_POINT_RESULT);
+        wireMockServer.stubFor(WireMock.delete(WireMock.urlMatching(resultEndPointUrl))
+            .willReturn(WireMock.ok()
+                .withHeader("Content-Type", "application/json; charset=utf-8")
+            ));
+        dynamicSimulationClient.deleteResults();
+
+        assertTrue(true);
```

Comment:
?

Reply:
It's to be homogeneous with the rest of the tests of this class, I'm not really sure how it's useful that said

Reply:
I'll remove it before merging